Core/PARStore.m

```diff
@@ -1579,7 +1595,10 @@ - (void)_sync
          {
              if (self._inMemoryCacheEnabled)
              {
-                 self._memory = [NSMutableDictionary dictionaryWithDictionary:updatedValues];
+                 [updatedValues enumerateKeysAndObjectsUsingBlock:^(id key, id newValue, BOOL *stop)
+                 {
+                     self._memory[key] = (newValue != [NSNull null] ? newValue : nil);
```

Comment:
Same question: Is removing the key the right answer here? We are discarding information that could be important to the caller.

Reply:
See above